proxy/snowflake.js

```diff
@@ -43,9 +45,9 @@ class Snowflake {
   // process. |pollBroker| automatically arranges signalling.
   beginWebRTC() {
     this.pollBroker();
-    return this.pollInterval = setInterval((() => {
-      return this.pollBroker();
-    }), this.config.defaultBrokerPollInterval);
+    return this.pollTimeout = setTimeout((() => {
+      return this.beginWebRTC()
```

Comment:
Hmm, should this really be `beginWebRTC()` and not `pollBroker()`?

Reply:
It should be `beginWebRTC()` since we want to call `setTimeout()` recusively.